teachertool/src/transforms/resetRubricAsync.ts

```diff
@@ -0,0 +1,24 @@
+import { stateAndDispatch } from "../state";
+import { isRubricLoaded } from "../state/helpers";
+import * as Actions from "../state/actions";
+import { setRubric } from "./setRubric";
+import { confirmAsync } from "./confirmAsync";
+import { makeRubric } from "../utils";
+import { Strings } from "../constants";
+
+export async function resetRubricAsync(fromUserInteraction: boolean) {
```

Comment:
What's the case that a rubric can be changed without user interaction?

Reply:
Good question! They can't, but it didn't feel safe to assume this transform would always be called in response to a user interaction. I was being defensive. Maybe overly so?

Reply:
Removed. We can always bring it back if necessary.